inference-engine/tests/functional/plugin/shared/src/single_layer_tests/lrn.cpp

```diff
@@ -36,15 +38,25 @@ std::string LrnLayerTest::getTestCaseName(testing::TestParamInfo<lrnLayerTestPar
 void LrnLayerTest::SetUp() {
     std::vector<size_t> inputShapes;
     auto netPrecision   = InferenceEngine::Precision::UNSPECIFIED;
-    size_t alpha, beta, bias, size;
-    std::tie(alpha, beta, bias, size, netPrecision, inputShapes, targetDevice) = GetParam();
+    double alpha, beta, bias;
+    size_t size;
+    std::string region;
+    std::tie(alpha, beta, bias, size, region, netPrecision, inputShapes, targetDevice) = GetParam();
 
     auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
     auto params = ngraph::builder::makeParams(ngPrc, {inputShapes});
     auto paramIn =
         ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
 
-    auto lrn = std::make_shared<ngraph::opset1::LRN>(paramIn[0], alpha, beta, bias, size);
+    std::vector<int64_t> axes;
+    if (region == "same") {
```

Comment:
I propose to parameterize the test directly with the `axes` parameter instead of some key-word which defines certain behavior of the test. 

There are two advantages to such an approach. First of all, it is to be aligned with opset specification which describes LRN parameters and inputs. And also to make it possible to validate cases different from currently possible axes values `{1}` and `{2,3,4,...}`

Reply:
In `ngraph::op::LRN` type of normalization channel algorithm is set using axes, but inside OpenVINO we use `ngraph::op::LRN_IE` with `std::string region`. Plugins have transformations to `ngraph::op::LRN_IE` and this op doesn't support `axes`. I don't quite undestand why we used `ngraph::op::LRN` in tests if plugins support `ngraph::op::LRN_IE` op.